baselines/pubsub-api-dump/api.json.baseline

```diff
@@ -0,0 +1,14065 @@
+{
+  "packageName": "google",
```

Comment:
This is the problem I will fix in the following PR: the package name must be `google.pubsub` but is detected incorrectly.